docs/examples/Rocket_Car.rst

```diff
@@ -0,0 +1,1585 @@
+    w_1_A2
+
+
+
+
+.. math::
+
+    \displaystyle 0
+
+
+
+.. code:: ipython3
+
+    w_2_A2 = N_w_A2.diff(u2, N)
+    w_2_A2
+
+
+
+
+.. math::
+
+    \displaystyle \frac{\operatorname{sin}\left(q_{4}\right)}{L}\mathbf{\hat{a_2}_z}
+
+
+
+:math:`\text{Accelerations and Angular Accelerations}`
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. code:: ipython3
+
+    a_1__P, a_2__P, a_3__P, a_1__C, a_2__C, a_3__C, a__Q, alpha__A1, alpha__A2 = sm.symbols('a_1__P, a_2__P, a_3__P, a_1__C, a_2__C, a_3__C, a__Q, alpha__A1, alpha__A2')
+
+.. code:: ipython3
+
+    N_a_P = N_v_P.dt(N).subs(soldict)
+    N_a_P
+
+
+
+
+.. math::
+
+    \displaystyle (- u_{1} u_{2} \operatorname{sin}\left(q_{4}\right) + \operatorname{cos}\left(q_{4}\right) \dot{u}_{2})\mathbf{\hat{a_2}_x} + \frac{u^{2}_{2} \operatorname{sin}\left(q_{4}\right) \operatorname{cos}\left(q_{4}\right)}{L}\mathbf{\hat{a_2}_y}
+
+
+
+.. code:: ipython3
+
+    N_a_C = N_v_C.dt(N).subs(soldict)
+    N_a_C
+
+
+
+
+.. math::
+
+    \displaystyle (- u_{1} u_{2} \operatorname{sin}\left(q_{4}\right) + \operatorname{cos}\left(q_{4}\right) \dot{u}_{2} - \frac{l u^{2}_{2} \operatorname{sin}^{2}\left(q_{4}\right)}{L^{2}})\mathbf{\hat{a_2}_x} + (\frac{l u_{1} u_{2} \operatorname{cos}\left(q_{4}\right)}{L} + \frac{l \operatorname{sin}\left(q_{4}\right) \dot{u}_{2}}{L} + \frac{u^{2}_{2} \operatorname{sin}\left(q_{4}\right) \operatorname{cos}\left(q_{4}\right)}{L})\mathbf{\hat{a_2}_y}
+
+
+
+.. code:: ipython3
+
+    N_a_Q = N_v_Q.dt(N).subs(soldict)
+    N_a_Q
+
+
+
+
+.. math::
+
+    \displaystyle \dot{u}_{2}\mathbf{\hat{a_1}_x} + \left(u_{1} + \frac{u_{2} \operatorname{sin}\left(q_{4}\right)}{L}\right) u_{2}\mathbf{\hat{a_1}_y}
+
+
+
+.. code:: ipython3
+
+    N_aa_A1 = N_w_A1.dt(N).subs(soldict)
+    N_aa_A1
+
+
+
+
+.. math::
+
+    \displaystyle (\dot{u}_{1} + \frac{u_{1} u_{2} \operatorname{cos}\left(q_{4}\right)}{L} + \frac{\operatorname{sin}\left(q_{4}\right) \dot{u}_{2}}{L})\mathbf{\hat{a_1}_z}
+
+
+
+.. code:: ipython3
+
+    N_aa_A2 = N_w_A2.dt(N).subs(soldict)
+    N_aa_A2
+
+
+
+
+.. math::
+
+    \displaystyle (\frac{u_{1} u_{2} \operatorname{cos}\left(q_{4}\right)}{L} + \frac{\operatorname{sin}\left(q_{4}\right) \dot{u}_{2}}{L})\mathbf{\hat{a_2}_z}
+
+
+
+:math:`\text{Forces}`
+~~~~~~~~~~~~~~~~~~~~~
+
+:math:`(F_r^*)_G = (F_r^*)_{GR} + (F_r^*)_{GI}`
+
+where,
+
+:math:`(F_r^*)_{GR} = {V_r}^G \cdot {F_G}^* + \omega_r^A \cdot {T_G}^*`
+
+:math:`F_G^* = -m_G {a^G}^*`
+
+:math:`T_G^* \overset{\Delta}{=} -[\alpha_A \cdot I_G + \omega_r^A \times (I_G \cdot \omega_r^A)]`
+
+:math:`({F_r}^*)_{GI} = -J\{\omega_r^A [\ddot{q_k} g_1 + \dot{q_k} (\omega_3^A g_2 - \omega_2^A g_3)] + C_{kr} (\dot{\omega_1^A} + \ddot{q_k}) \}`
+
+#####
+
+.. raw:: html
+
+   <center>
+
+:math:`\text{Kane, T.R., 1978. Nonholonomic multibody systems containing gyrostats. In Dynamics of Multibody Systems (pp. 97-107). Springer, Berlin, Heidelberg.}`
+
+.. raw:: html
+
+   </center>
+
+:math:`\text{Naming Convention:}`
```

Comment:
All of these are in math directives for some reason. All normal text needs to be extracted from these and use proper rst notation.